Cargo.toml

```diff
@@ -46,13 +45,16 @@ paste = "1"
 dsp = { path = "dsp" }
 ad9959 = { path = "ad9959" }
 
+[dependencies.miniconf]
+git = "https://github.com/vertigo-designs/miniconf.git"
+branch = "develop"
+
 [dependencies.mcp23017]
 git = "https://github.com/mrd0ll4r/mcp23017.git"
 
-[dependencies.smoltcp]
-version = "0.7"
-features = ["ethernet", "proto-ipv4", "socket-tcp", "proto-ipv6"]
-default-features = false
+[dependencies.smoltcp-nal]
+git = "https://github.com/vertigo-designs/smoltcp-nal.git"
```

Comment:
Update git dependencies to the quartiq repositories - also, publish `smoltcp-nal`